package.json

```diff
@@ -12,6 +12,8 @@
     "@makerdao/dai-plugin-ledger-web": "^0.9.7",
     "@makerdao/dai-plugin-trezor-web": "^0.9.6",
     "@makerdao/ui-components": "1.0.0-alpha.29",
+    "@makerdao/ui-components-core": "^0.1.4",
```

Comment:
This is unfortunate...but if we want to get the release out soon we can add this and slowly remove the old version of ui-components